examples/blog/package.json

```diff
@@ -14,10 +14,11 @@
     "graphql": "^14.3.1",
     "graphql-yoga": "^1.18.2",
     "nexus": "^0.12.0-beta.8",
-    "nexus-prisma": "file:../.."
+    "nexus-prisma": "^0.4.1"
```

Comment:
More realistic and will catch more user issues. For example ones like this. And we can still `yarn link nexus-prisma` these when we want locally. The thing that is hard (impossible?) to reproduce with yarn/npm (link, file, ...) is things like the effect of the `package.json` `files` entry.